src/components/Tags/TagsComponent.tsx

```diff
@@ -87,6 +93,16 @@ export default function TagsComponent({ mode, gameData, setGameData }: Props) {
     <ChakraProvider theme={chakraTheme}>
       <div>
         <div className="m-auto flex w-5/6 flex-row flex-wrap pb-3 pt-6 font-inter text-base">
+          {gameData.videoTrailer ? <Tag>Video Trailer</Tag> : null}
+          {gameData.parentingGuide ? <Tag>Parenting Guide</Tag> : null}
+          {gameData.lesson ? <Tag>Lesson Plan</Tag> : null}
+          {gameData.answerKey && admin ? <Tag>Answer Key</Tag> : null}
```

Comment:
The answer key should be visible to admin, parents, and educators (not just admin)